src/TetherComponent.jsx

```diff
@@ -286,4 +291,18 @@ class TetherComponent extends Component {
   }
 }
 
+function componentWillUpdate(nextProps) {
```

Comment:
wish we could use `componentDidUpdate` but I don't think it's viable for React 15.x due to the way it'll render. We may want to test it, but I think this solution works for now